columnq/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/lib.rs"
 arrow-schema = { version ="26", features = ["serde"] }
 
 datafusion = "14"
-object_store = "0.5.0"
+object_store = { version = "0.5.3", features = ["aws_profile"] }
```

Comment:
the feature `aws_profile` was introduced since `0.5.2`. I set minimum to `0.5.3` as it is latest one as of today

https://github.com/apache/arrow-rs/blob/master/object_store/CHANGELOG-old.md#object_store_052-2022-12-02